src/core/client/stream/tabs/Comments/Stream/FeaturedComments/FeaturedCommentContainer.tsx

```diff
@@ -72,6 +74,10 @@ const FeaturedCommentContainer: FunctionComponent<Props> = (props) => {
     [emitViewConversationEvent, comment.id, setCommentID]
   );
 
+  const featuringUser = comment.tags.find(
+    (tag) => tag.code === "FEATURED"
```

Comment:
Can you please update to use `GQLTAG.FEATURED` instead of `"FEATURED"` here?

Reply:
✅